### PR TITLE
Production Fixes

### PR DESCRIPTION
**Pull request by seveibar**

### CODE REVIEW

File: client/app/courses/view/courseview.html [hunk 1]
```diff
@@ -1,113 +1,111 @@
...
-          <h4> Sections </h4>
-          <table class="course-section text-center table table-hover">
-            <tr>
-              <th class="text-center">Number</th>
-              <th class="text-center">Instructor</th>
-              <th class="text-center">Students</th>
-              <th class="text-center">Enrollment Policy</th>
-              <th class="text-center">Info</th>
-              <th class="text-center">Enrolled</th>
+                <table class="course-section text-center table table-hover">
```
Comment (kmcnellis) [inline client/app/courses/view/courseview.html:47]: Can you revert these spacing changes? very hard to see actual changes

File: client/components/courseTitleBanner/courseTitleBannerBase.html [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
-      </span>
-      <span>| Instructor<span ng-show="section.instructors.length > 1">s</span>:
-        <span ng-repeat="instructor in section.instructors">
-          <span ng-show="$last && !$first">& </span>{{instructor.firstName}} {{instructor.lastName}}
-          <span ng-hide="$last">, </span>
+    <h2 class="col-sm-12"> {{ course.name }}</h2>
+    <div class='course-number col-sm-10'>{{course.department}} {{course.courseNumber}}
+        <span ng-show="section">| Section{{section.sectionNumbers.length > 1 ? "s": ""}}:
+            <span ng-repeat="number in section.sectionNumbers track by $index" ng-show="number > -1">
+                <span ng-show="$last && !$first">& </span>{{number}}<span ng-hide="$last || section.sectionNumbers.length < 3">, </span>
```
Comment (kmcnellis) [inline client/components/courseTitleBanner/courseTitleBannerBase.html:7]: And these. Why are they all 4 spaces now?

File: client/components/sectionform/sectionform.html [hunk 3]
```diff
@@ -1,60 +1,60 @@
 <div class='row'>
-  <div class='col-xs-12'>
-    <h4> Edit Section </h4>
+    <div class='col-xs-12'>
```
Comment (kmcnellis) [inline client/components/sectionform/sectionform.html:2]: and here

File: .htmlhintrc [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
+    "attr-bans": [],
```
Comment (kmcnellis) [inline .htmlhintrc:2]: Don't think we need a file version of this if its in grunt

File: package.json [hunk 5]
```diff
@@ -74,6 +74,8 @@
     "grunt-express-server": "^0.5.1",
     "grunt-filerev": "^2.3.1",
     "grunt-google-cdn": "~0.4.0",
+    "grunt-htmlhint": "^0.9.13",
+    "grunt-htmllint": "^0.2.7",
```
Comment (kmcnellis) [inline package.json:78]: Why do we need both of these?

File: .htmlhintrc (hunk 4 above)
Comment (seveibar) [inline .htmlhintrc:2]: it's for IDEs

File: package.json (hunk 5 above)
Comment (seveibar) [inline package.json:78]: they each play differently w/ angular directives

**Comment by kmcnellis** [comment]
Broken: 
screen shot 2016-08-01 at 5 45 42 pm

File: client/app/courses/view/courseview.html (hunk 1 above)
Comment (seveibar) [inline client/app/courses/view/courseview.html:47]: ehh it just fixes tags- i don't think it really needs review

File: client/components/courseTitleBanner/courseTitleBannerBase.html (hunk 2 above)
Comment (seveibar) [inline client/components/courseTitleBanner/courseTitleBannerBase.html:7]: if we define a standard ill use that, but we have to come to an agreement on it

**Comment by seveibar** [comment]
hence why this is marked "needs fixing"

**Comment by kmcnellis** [comment]
Broken: 
screen shot 2016-08-01 at 5 58 27 pm

**Comment by kmcnellis** [comment]
Just documenting :)

**Comment by seveibar** [comment]
ahh okay thanks thats useful actually

**Comment by kmcnellis** [comment]
n/a

**Comment by seveibar** [comment]
@kmcnellis that's intentional- those events are expired. @joey introduced that a while ago

**Comment by kmcnellis** [comment]
Then we should hide them or put an "old" tag :/ (Haven't looked at that page for a while)

**Comment by seveibar** [comment]
Yea I agree- filed as #364

**Comment by seveibar** [comment]
@kmcnellis do you want to take a swing at this issue- im actually a bit stuck

**Comment by seveibar** [comment]
I thought I fixed it by correcting the HTML- but it seemed to have started messing up again

**Comment by kmcnellis** [comment]
What are you seeing?
I'm trying to tackle the image urls really quick

**Comment by seveibar** [comment]
I see the same thing as you- np I'll keep at it.

**Comment by kmcnellis** [comment]
How is this going?

**Comment by kmcnellis** [comment]
Fixed some things. Check it out now?

**Comment by seveibar** [comment]
awesome thanks kiana- reviewing

**Comment by seveibar** [comment]
awesome!!!

**Pull request merged**